docs/source/build/index.rst

```diff
@@ -241,12 +221,10 @@ Run in terminal:
 .. code-block:: shell
 
   git clone https://github.com/hyperledger/iroha.git
-  iroha/vcpkg/build_iroha_deps.sh
-  vcpkg-build/vcpkg integrate install
+  cd iroha
+  ./vcpkg/build_iroha_deps.sh
```

Comment:
@kuvaldini I am not sure it will not be confusing as in Docker it is just the vcpkg thing -- and with others it is prerequisites and then vcpkg -- and with navigation directly to the platform at the top of the page it might not be the best option, I am afraid. 
I added the directory, of course, thank you!

Reply:
@kuvaldini please let me know if you do not agree on the scenario thing. Otherwise the PR is updated